src/main/java/com/minecolonies/entity/EntityCitizen.java

```diff
@@ -536,6 +598,29 @@ private double getExperiencePoints()
         return citizenData.getExperience();
     }
 
+    /**
+     * Updates the armour damage after being hit.
+     *
+     * @param damage damage dealt.
+     */
+    private void updateArmorDamage(double damage)
+    {
+        for (ItemStack stack : this.getArmorInventoryList())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'stack' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)